extract-files.sh

```diff
@@ -15,9 +15,12 @@ mkdir -p $BASE/$DIR
 adb pull /system/$FILE $BASE/$FILE
 done
 
-# some extra stuff
+# Pull extra stuff from FP OS
 adb pull /system/lib/hw/audio_policy.default.so $BASE/lib/hw/audio_policy.mt6589.so
-adb pull /system/lib/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
+adb pull /system/lib/hw/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
```

Comment:
in `/system/lib/hw` to only files are

>-rw-r--r-- root     root        13604 2008-08-01 14:00 libaudio.r_submix.default.so
-rw-r--r-- root     root         9488 2008-08-01 14:00 libaudio.usb.default.so

, BUT `libaudio.primary.default.so` is in `/system/lib`